.travis.yml

```diff
@@ -1,29 +1,19 @@
+dist: bionic
 language: python
-dist: bionic   # required for Python >= 3.7
 cache: pip
-
-# Environment changes have to be manually synced with 'tox.ini'.
-# See: https://github.com/travis-ci/travis-ci/issues/3024
```

Comment:
Why is this comment removed? Again, it was there for a reason.

Reply:
That issue was closed in 2015.

Reply:
That's irrelevant. Both files still list the same environments, and should still be kept in sync. If that's no longer necessary, only one of the files should have the list of environments, not both. Either properly handle this (in a separate commit, as it has nothing to do with getting flake8 to run in CI) or don't touch it.